.gitignore

```diff
@@ -11,11 +12,6 @@
 /.project
 /.pydevproject
 /TAGS
-/Tribler/Test/data/*.sdb
-/Tribler/Test/test_.Tribler/
-/Tribler/Test/test_.Tribler2
-/Tribler/Test/test_TriblerDownloads2
-/Tribler/community/tunnel/.Tribler*
```

Comment:
You shouldn't keep state data in the code directories. Move it to the home dir or make it a parameter.